### DISCUSSION TITLE
Make it hotreloadable?

### DISCUSSION THREAD

**Discussion by SuperCipher** [sanhapon/drawing#16] [Ideas]
Right now I was thinking able using [nodemon](https://www.npmjs.com/package/nodemon) since I'm familiar with it. But not sure if there's a better way?
I also saw [cargo-watch](https://github.com/watchexec/cargo-watch) but I have never used it.
What's your thought on this @sanhapon

**Comment by sanhapon** [comment]
I'm afraid that wasm-pack, which I'm using, has no watch command. It's irritating to make change and build at this time. If I can find the better way or able to integrate with nodemon, will post it here.
